src/app/templates/app/convert.html

```diff
@@ -138,41 +133,52 @@ <h4 id="doclabel">Upload SPDX Document using the button or by dragging and dropp
   $("#convertpage").addClass('linkactive');
 
   $("#from_format").change(function() {
-    $("#tagToRdfFormat").prop("disabled",'disabled');
-    $(".tagToRdfOptions").css("color",'grey');
     if ($(this).data('options') === undefined) {
       /*Taking an array of all options-2 and kind of embedding it on the select1*/
       $(this).data('options', $('#to_format option').clone());
     }
     var id = $(this).val();
-    var options = $(this).data('options').filter('[data-value=' + id + ']');
+	var id2 = $('#to_format').val();
```

Comment:
nit: indent, although it doesn't matter in js/html

Reply:
A bit of a problem with tabs - fixed in commit 981e0d5